src/pages/components/toast.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import toastContent from '../../../locale/fi/toast.json';
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import sideNavData from 'config/sidenav/components';
+import NoteBox from 'components/NoteBox';
+import { Toast } from 'components/ExampleComponents';
+import { Heading, Text, Paragraph } from 'components/ResponsiveComponents';
+import Section from 'components/Section';
+import ComponentDescription from 'components/ComponentDescription';
+import ComponentExample from 'components/ComponentExample';
+import { defaultSuomifiTheme } from 'suomifi-ui-components';
```

Comment:
This import is better to be located after the import of React. First external imports and then internal ones.

Reply:
Fixed place of imports